packages/config/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tsconfig.json"
```

Comment:
If I were doing things properly, I would be referencing a package for these. Check out `npx create-turbo@latest` for an example of how to do it!

Reply:
I was thinking about looking into Turborepor since I saw its name shine down on me while browsing Vercel. Thank you for the heads up again.